### DISCUSSION TITLE
Supabase RLS for _prisma_migrations

### DISCUSSION THREAD

**Discussion by Jordaneisenburger** [supabase/supabase#14152] [General]
Hi all,

I'm fairly new to Supabase and I noticed that when using Prisma, RLS is disabled for the `_prisma_migrations` table. According to the hints inside the Supabase dashboard this means that anyone can request and modify this table?

I assume enabling RLS is the right choice but that would also mean that Prisma won't be allowed to make changes anymore unless I make a policy for it (which I don't know how to). Any guidance/advice here would be really appreciated! 

Thanks!

**Comment by Jordaneisenburger** [comment]
I'm stupid

**Reply by dmaslan** [reply]
Hey @Jordaneisenburger I was wondering the same thing (i'm a noob to supabase and databases in general). Can you enlighten me?

**Reply by Jordaneisenburger** [reply]
If set up correctly only the postgress role will be able to access it, if you look at the supabase dashboard and click into the prisma schema you can switch roles to see what anon users would see, which is nothing
